tests/test_request_scoped_k8s_integration.py

```
@@ -0,0 +1,127 @@
+        assert f"token:{user}" in data, f"{user} did not get its own token"
+        # No other user's identity may appear in this invocation's output.
+        others = [u for u in users if u != user and f"token:{u}" in data]
+        assert not others, f"{user} saw foreign identities: {others}"
+
+
+def test_no_global_state_mutation_and_no_leftover_files(
+    fake_kubectl, ca_file, monkeypatch
+):
+    monkeypatch.setenv(AUTH_MODE, "request_token")
+    before_env = dict(os.environ)
+    tmpdir = os.environ.get("TMPDIR", "/tmp")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)